### DISCUSSION TITLE
Address binary time series prediction with highly imbalance dataset

### DISCUSSION THREAD

**Discussion by Jieyi1114** [awslabs/gluonts#3177] [Q&A]
I'm using DeepAREstimator  to address a multiple binary time series problem. My target variable is highly imbalance (30:1).  I set the threshold manually transforming the outputs to [0,1], but the performance is unsatisfactory and unstable.Can I modify the loss function to change the weights? 

My current code is:
`

mx.random.seed(1)
np.random.seed(1)
prediction_length = 9
estimator = DeepAREstimator(
    freq='W',
    prediction_length = prediction_length, 
    context_length = 8,
    use_feat_dynamic_real=True,
    use_feat_static_real=True,
    use_feat_static_cat=True,
    cardinality=[26, 6, 2, 21],  # Adjust cardinality values as needed
    cell_type='lstm',
    num_layers = 5,
    num_cells = 40,
    trainer = Trainer(epochs = 30, learning_rate = 0.01))
predictor = estimator.train(train_ds)
forecast_it, ts_it = make_evaluation_predictions(dataset=test_ds, predictor=predictor, num_samples=10)
...

`
